src/Db/PdoRepository.php

```diff
@@ -21,10 +21,10 @@ class PdoRepository
      * @param PDO $pdo An open database connection
      * @param string $table Table name where Xhgui profiles are stored
      */
-    public function __construct(PDO $pdo)
+    public function __construct(PDO $pdo, string $table)
     {
         $this->pdo = $pdo;
-        $this->tableResults = 'results';
+        $this->tableResults = sprintf('"%s"', $table);
```

Comment:
makes sense to be symmetric and have watches table configurable. please add similar changes for the watches table.